### PR TITLE
Add Procreate icon

### LINKED ISSUE #7944: Request: Procreate Icon
### Brand Name

Procreate

### Website

https://procreate.art

### Popularity Metric

The Similarweb rank is 55,442. See https://www.similarweb.com/website/procreate.com

### Official Resources for Icon and Color

None

### Additional Comments

App icon can be found here: https://assets.procreate.art/img/procreate-icon.png

There's a press kit but no official logos that I can find. I would presume the "feather" symbol in the app icon is what we're after.

Press Kit: https://procreate.art/press

### Contributing

No

### PR DESCRIPTION
**Pull request by LeviSnoot**
**Issue:** closes #7944 

**Similarweb rank:** 1,543,095

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Logo was vectorized manually, and 0.5px gaps were added to distinguish between the elements in the "feather" as per `CONTRIBUTING.md`. Procreate doesn't provide any branding guidelines despite having a press/media kit available, so the "brand color" was color picked from the logo itself. I selected one of the most prominent colors in the logo for this.

procreate

### CODE REVIEW

**Comment by LeviSnoot** [comment]
Actually very confused about what the linter and verofier are complaining about. If someone can guide me here that'd be much appreciated!

File: icons/procreate.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Procreate</title><path d="m11.33 15.84-1.35.35-.31.07c-2.71.64-4.81 1.16-6.39 2.2l.1-.1a7.32 7.32 0 0 1 3.2-1.94c.94-.25 2.19-.35 3.52-.47l1.24-.11m2.11-6.72c-.82.21-1.71.42-2.64.69a16.54 16.54 0 0 0-6.14 3.09 12.37 12.37 0 0 0-3.68 5.47 14.97 14.97 0 0 1 9.81-7.96c.93-.42 1.82-.86 2.68-1.31m5.36 3.3c-1.7.83-3.58 1.15-5.53 1.48-2.15.37-4.36.74-6.51 1.84l-.4.23.08-.02c1.9-.5 4.79-.44 7.63-.97h.03a17.7 17.7 0 0 0 4.71-2.55m3.81-9.06a11.66 11.66 0 0 1-3.27 3.15c-.88.61-1.85 1.09-2.96 1.52a43.57 43.57 0 0 1-3.51 1.94c1.92-.39 5.51-1.23 7.28-2.72a8.16 8.16 0 0 0 2.46-3.9m-.71 5.82c-.25.36-.54.71-.86 1.04-3.79 3.87-9.27 2.42-14.51 5.08A13.9 13.9 0 0 0 3.08 18a11.03 11.03 0 0 0-2.94 5.31v.06c-.04.16-.06.33-.08.49v.13H.05L.1 23a14.67 14.67 0 0 1 12.12-12.38c2.26-.43 6.29-1.29 8.3-2.98 2-1.68 3.13-4.56 3.14-7.6V0c.05.35.7 5.58-1.72 9.18Z" style="fill:#fff;stroke-width:0"/></svg>
```
Comment (service-paradis) [inline icons/procreate.svg:1]: I did not review the icon yet, but the linter is failing due to the extra attribute `style`. We need to remove it:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Procreate</title><path d="m11.33 15.84-1.35.35-.31.07c-2.71.64-4.81 1.16-6.39 2.2l.1-.1a7.32 7.32 0 0 1 3.2-1.94c.94-.25 2.19-.35 3.52-.47l1.24-.11m2.11-6.72c-.82.21-1.71.42-2.64.69a16.54 16.54 0 0 0-6.14 3.09 12.37 12.37 0 0 0-3.68 5.47 14.97 14.97 0 0 1 9.81-7.96c.93-.42 1.82-.86 2.68-1.31m5.36 3.3c-1.7.83-3.58 1.15-5.53 1.48-2.15.37-4.36.74-6.51 1.84l-.4.23.08-.02c1.9-.5 4.79-.44 7.63-.97h.03a17.7 17.7 0 0 0 4.71-2.55m3.81-9.06a11.66 11.66 0 0 1-3.27 3.15c-.88.61-1.85 1.09-2.96 1.52a43.57 43.57 0 0 1-3.51 1.94c1.92-.39 5.51-1.23 7.28-2.72a8.16 8.16 0 0 0 2.46-3.9m-.71 5.82c-.25.36-.54.71-.86 1.04-3.79 3.87-9.27 2.42-14.51 5.08A13.9 13.9 0 0 0 3.08 18a11.03 11.03 0 0 0-2.94 5.31v.06c-.04.16-.06.33-.08.49v.13H.05L.1 23a14.67 14.67 0 0 1 12.12-12.38c2.26-.43 6.29-1.29 8.3-2.98 2-1.68 3.13-4.56 3.14-7.6V0c.05.35.7 5.58-1.72 9.18Z"/></svg>
```

File: _data/simple-icons.json [hunk 2]
```diff
@@ -11857,6 +11857,11 @@
             "hex": "2480E6",
             "source": "https://processwire.com"
         },
+        {
+            "title": "Procreate",
+            "hex": "E559D7",
+            "source": "https://procreate.art"
+        },
```
Comment (service-paradis) [inline _data/simple-icons.json:11864]: Here the linter is failing due to the extra spaces after the comma:
```suggestion
        {
            "title": "Procreate",
            "hex": "E559D7",
            "source": "https://procreate.art"
        },
```

**Review by service-paradis — CHANGES_REQUESTED** [review]
Thank you @LeviSnoot for your contribution. I did not review your icon yet, but I left a few comments about the action failures.

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
@LeviSnoot, the linting is now failing because the icon isn't _exactly_ 24dp wide and is slightly off centre on both axes. Both of these could likely just be due to the precision being set too low when optimising.

I also think we should be including the background square as it looks to me like its consistently used as part of their icon. What do you think, @service-paradis?

**Comment by LeviSnoot** [comment]
> `@LeviSnoot`, the linting is now failing because the icon isn't _exactly_ 24dp wide and is slightly off centre on both axes. Both of these could likely just be due to the precision being set too low when optimising.

I noticed this after making the previously requested changes, so I cloned it locally and tried fixing things there so I'm not spamming a bunch of commits that end up failing. I can't seem to fix it though because even when I made sure the dimensions are exactly correct and centered in Adobe Illustrator, the linter still complained about centering:

```
x icons\procreate.svg
  x icon-centered <path> must be centered at (12, 12); the center is currently (11.858, 12)
```

So I'm stuck. If anyone would like to inspect the SVG in its current state and help me figure it out, maybe using a different tool than Illustrator, that would be very much appreciated, I'll attach it below:

[procreate.svg](https://github.com/simple-icons/simple-icons/assets/13402525/8f3b7118-9cba-41f9-aafb-9e919973b81f)


> 
> I also think we should be including the background square as it looks to me like its consistently used as part of their icon. What do you think, `@service-paradis`?

I know I'm not the one being asked but I'd like to make a couple of arguments against this if that's alright.

- Savage (the company behind Procreate) has another product called "Procreate Pocket", which is a slimmed down version of the app intended for iPhone use. In this app the feather glyph remains the exact same shape with the main difference being that the "background square" is white instead of dark grey.
- I believe the only reason they use the rounded rectangle is to comply with the App Store guidelines, and the glyph itself is what people recognize as the logo for Procreate. Since Procreate is an Apple exclusive suite of apps, there's less reason for Savage to publish the icon by itself.
- Adding a rectangle would introduce margins to the logo, which I feel is already kind of busy with the amount of lines going through it. Reducing the size of the glyph while keeping the strokes 0.5px wide may make it hard to read.

Thank you for your thoughtful feedback @PeterShaggyNoble. I'll hold off on further commits until I hear back.

**Comment by service-paradis** [comment]
Brand kit and logo guidelines are available at https://procreate.com/brand-use

According to their PDF guidelines, we need to use the background frame:
image

Unfortunately, we cannot add this logo according to their guidelines:

> Do not alter or modify in any way
> Do not recolour, add filters, or change the hue, tone, or brightness of the App Icon.

**Pull request closed without merging**